wad/__main__.py

```diff
@@ -48,7 +48,7 @@ def main(timeout=TIMEOUT):
     parser.add_option("-o", "--output", dest="output_file", metavar="FILE",
                       help="output file for detection results (default: STDOUT)")
 
-    parser.add_option("-c", "--clues", dest="clues_file", metavar="FILE", default=CLUES_FILE,
+    parser.add_option("-c", "--clues", dest="clues_file", metavar="FILE", default=CLUES_FILE_PATHS[0],
```

Comment:
The idea is to try to load from multiple places by default (first $pwd/etc/apps.json then /etc/wad/apps.json) rather than loading multiple clues files (https://github.com/CERN-CERT/WAD/pull/22/files/e83859194e58947b73e6a59f7e7cc16b23baaf4d#diff-1b57a6d4f68d48b9c21c39247beeb114R96). So if you run wad as a python script it finds the apps.json through the relative path and if you install it apps.json gets installed to /etc/wad/apps.json.

Reply:
I understand the idea behind it, but the way the __main__.py file is using it remove all the functionality doesn't it?
`Clues.get_clues` will be either called with `CLUES_FILE_PATHS[0]` if no options is passed or with the single file passed over, isn't?

Reply:
Yes, you're right. I see what you mean. The script we use for scanning internally causes `get_clues` to get called without parameters which works as I expected, but when executing the binary in this repo directly with a parameter it's not handled properly.
Fixed now.